server/src/graphql/mutation_root.rs

```diff
@@ -40,4 +40,14 @@ impl MutationRoot {
             .await
             .or_else(|_e| Err(FieldError::from("Failed to add person to squad")))
     }
+
+    async fn new_transaction(
+        &self,
+        context: &Context<'_>,
+        input: NewTransactionInput,
+    ) -> FieldResult<NewTransactionPayload> {
+        new_transaction(context.data::<Pool>(), input.try_into()?)
```

Comment:
Looking forward - when we add an 'edit transaction' button, what do we want to do on the back end? Will we create a new transaction and delete the old one or just do an update operation and keep the same node id

Reply:
That's a good question... I like the idea of having a new transaction (since we could expose the version history of each transaction), but I feel like it would become difficult to separate out the obsoleted transactions.

Maybe we could have Transaction consist of multiple TransactionVersions, where each TransactionVersion has its own TransactionParts? But then we'd still need to separate out the obsoleted TransactionParts :-\